.github/workflows/upload-pypi-source.yml

```diff
@@ -194,6 +199,7 @@ jobs:
 
       - name: Build source distribution
         run: |
+          # TODO: let poetry do this?
```

Comment:
Using `-m build` is correct, though we should not pass `--sdist` in general and let it build both a wheel and sdist and publish both.  That's for another time though.

```suggestion
```